stagemonitor-core/src/main/java/org/stagemonitor/core/CorePlugin.java

```diff
@@ -396,24 +395,17 @@
 			.configurationCategory(CORE_PLUGIN_NAME)
 			.tags("elasticsearch", "advanced")
 			.buildWithDefault(5);
-	private final ConfigurationOption<Collection<String>> remotePropertiesConfigUrls = ConfigurationOption.stringsOption()
+	private final ConfigurationOption<List<URL>> remotePropertiesConfigUrls = ConfigurationOption.urlsOption()
 			.key("stagemonitor.configuration.remoteproperties.urls")
 			.dynamic(false)
 			.label("URLs of the remote properties")
-			.description("Must be http or https urls.")
+			.description("Must be http or https URLs. This can be a single URL or a list of config URLs." +
```

Comment:
Don't forget the white space at the end of each sentence.

Reply:
Added whitespaces